tempesta_fw/http_parser.c

```diff
@@ -70,7 +83,8 @@ int __fsm_const_state;							\
 int __fsm_n __attribute__((unused));					\
 size_t __fsm_sz __attribute__((unused));				\
 unsigned char *__fsm_ch __attribute__((unused));			\
-TfwStr *__fsm_str __attribute__((unused));
+TfwStr *__fsm_str __attribute__((unused));				\
```

Comment:
This variable is unused in the code now. Perhaps it's no harm to keep it anyway, but maybe it's better to remove it altogether.